packages/data-point/lib/helpers/type-check-function-reducers.js

```diff
@@ -2,11 +2,14 @@ const Util = require('util')
 const _ = require('lodash')
 const utils = require('../utils')
 
-function errorMessage (value, expectedType) {
-  const entityId = _.get(value, 'reducer.spec.id', 'value')
```

Comment:
I'm not sure if this line was needed, but it looks like it expects `value` to be an accumulator, which is not the case